codexdht.nimble

```diff
@@ -1,24 +1,24 @@
 # Package
 
-version       = "0.4.0"
+version       = "0.5.0"
 author        = "Status Research & Development GmbH"
 description   = "DHT based on Eth discv5 implementation"
 license       = "MIT"
 skipDirs      = @["tests"]
 
 # Dependencies
 requires "secp256k1#2acbbdcc0e63002a013fff49f015708522875832" # >= 0.5.2 & < 0.6.0
-requires "protobuf_serialization" # >= 0.2.0 & < 0.3.0
+requires "protobuf_serialization#5a31137a82c2b6a989c9ed979bb636c7a49f570e" # >= 0.2.0 & < 0.3.0
 requires "nimcrypto >= 0.5.4"
 requires "bearssl == 0.2.5"
 requires "chronicles >= 0.10.2 & < 0.11.0"
 requires "chronos >= 4.0.3 & < 4.1.0"
 requires "libp2p == 1.5.0"
-requires "metrics"
-requires "stew#head"
-requires "stint"
-requires "https://github.com/codex-storage/nim-datastore >= 0.1.1 & < 0.2.0"
-requires "questionable"
+requires "metrics#cacfdc12454a0804c65112b9f4f50d1375208dcd"
+requires "stew >= 0.2.0"
```

Comment:
Usually a good idea to prevent major versions (with breaking changes) from creeping in 
```suggestion
requires "stew >= 0.2.0 & < 0.3.0"
```